lua/telescope/pickers/layout_strategies.lua

```diff
@@ -806,13 +807,22 @@ layout_strategies.bottom_pane = make_documented_layout(
     results.height = height - prompt.height - (2 * bs)
     preview.height = results.height - bs
 
+    local width_opt = layout_config.width
+    local width = resolve.resolve_width(width_opt)(self, max_columns, max_lines)
+
```

Comment:
Since `bottom_pane` always takes up the full width of the screen, we shouldn't have an option for it

Reply:
local width = max_columns?

Reply:
I don't think there is any need to set `local width = max_columns`, since we already have a variable with that value, and we don't use the value of `width` again until after it is recalculated.
It's better to just introduce the local variable (probably at the same time as `w_space`).

Reply:
afaict I can also remove width_opt then (ls said it's unused then)